data_pipeline/algolia-update.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs');
+const dotenv = require('dotenv');
+const algoliaSearch = require('algoliasearch');
+
+//load local environment variables
+const env = {
+  local: dotenv.parse(fs.readFileSync('.env.local'))
+};
+
+const charities = JSON.parse(
+  fs.readFileSync('data_pipeline/charity.json').toString('utf8')
+);
+
+const updatedAlgoliaIndex = Object.entries(charities).map(
+  ([charityID, { charityname, category, servicetype }]) => {
+    return {
```

Comment:
can shorthand this

Reply:
[upderped](https://github.com/CodeForBoulder/upswyng/pull/74/commits/0fe944f5aed3d51015382d6fdea052a981c9a248)